servicetalk-client-api-internal/src/main/java/io/servicetalk/client/api/internal/DefaultPartitionedClientGroup.java

```diff
@@ -74,8 +75,7 @@
          * @param psd the partitioned {@link ServiceDiscoverer}
          * @return new client for the given arguments
          */
-        Client apply(PartitionAttributes pa,
-                     ServiceDiscoverer<U, R, ? extends PartitionedServiceDiscovererEvent<R>> psd);
+        Client apply(PartitionAttributes pa, ServiceDiscoverer<U, R, ServiceDiscovererEvent<R>> psd);
```

Comment:
We can also kill `? extends ` in ctor argument (line 101) and in `GroupedByPartitionSubscriber` (lines 258, 278).